src/main/java/mate/academy/model/dto/request/MovieRequestDto.java

```diff
@@ -1,6 +1,9 @@
 package mate.academy.model.dto.request;
 
+import javax.validation.constraints.NotNull;
+
 public class MovieRequestDto {
+    @NotNull
```

Comment:
the annotation has no sense if you won't add the annotation @Value into your controller methods.
the same is about all of your request dtos, check them all

Reply:
@Valid *